website/docs/d/system_center_virtual_machine_manager_inventory_items.html.markdown

```diff
@@ -0,0 +1,47 @@
+---
+subcategory: "System Center Virtual Machine Manager"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_system_center_virtual_machine_manager_inventory_items"
+description: |-
+  Gets information about existing System Center Virtual Machine Manager Inventory Items.
+---
+
+# Data Source: azurerm_system_center_virtual_machine_manager_inventory_items
+
+Use this data source to access information about existing System Center Virtual Machine Manager Inventory Items.
+
+## Example Usage
+
+```hcl
+data "azurerm_system_center_virtual_machine_manager_inventory_items" "example" {
+  system_center_virtual_machine_manager_server_id = azurerm_system_center_virtual_machine_manager_server.example.id
+}
+```
+
+## Argument Reference
+
+* `system_center_virtual_machine_manager_server_id` - The ID of the System Center Virtual Machine Manager Server.
+
+## Attributes Reference
+
+* `id` - The ID of the System Center Virtual Machine Manager Inventory Items.
+
+* `inventory_items` - One or more `inventory_item` blocks as defined below.
```

Comment:
The property name in the schema is `inventory_items` so that should be used everywhere
```suggestion
* `inventory_items` - One or more `inventory_items` blocks as defined below.
```

Reply:
Updated